apps/sharing/tests.py

```diff
@@ -51,10 +51,10 @@ class SharingModelsTestCase(BaseTestCase):
     def test_share_count(self):
         addon = Addon.objects.get(id=3615)
 
-        eq_(addon.share_counts[DIGG.shortname], 29)
+        eq_(addon.share_counts[TWITTER.shortname], 29)
 
         # total count with no shares
-        eq_(addon.share_counts[FACEBOOK.shortname], 0,
+        eq_(addon.share_counts[TWITTER.shortname], 0,
```

Comment:
Revert this line to what it was at first, and remove the lines 2-10 in the file `apps/sharing/fixtures/sharing/share_counts.json` as explained earlier, it will work